mc.js

```diff
@@ -207,8 +207,8 @@ mix(Mc.prototype,{
       get value(){
         return buf;
       },
-      set value(val,mode){
-        buf = mode=='append'?buf+val:(mode=='prepend'?val+buf:val);
+      set value(val){
```

Comment:
this seems to remove support for memcache append and prepend commands. is this intentional or handled in a different way i dont see?